traffic_ops/app/lib/API/DeliveryService/SslKeys.pm

```diff
@@ -161,30 +172,42 @@ sub view_by_hostname {
 			return $self->alert( { Error => " - A delivery service does not exist for a host with hostanme of $key" } );
 		}
 
-		my $xml_id = $ds->xml_id;
+		my $ds_id = $ds->id;
 
 		if ( !$version ) {
 			$version = 'latest';
 		}
-		$key = "$xml_id-$version";
+		$key = "ds_$ds_id-$version";
```

Comment:
As discussed in Slack, xml_id is immutable now, will revert this change in TC-547.